src/uss/ussNodeActions.ts

```diff
@@ -91,7 +92,7 @@ export async function createUSSNodeDialog(node: ZoweUSSNode, ussFileProvider: US
     }
 }
 
-export async function deleteUSSNode(node: ZoweUSSNode, ussFileProvider: USSTree, filePath: string) {
+export async function deleteUSSNode(node: IZoweTreeNode, ussFileProvider: IZoweTree<IZoweTreeNode>, filePath: string) {
```

Comment:
Same question here, should these set of actions be part of the corresponding class?

Reply:
Yes did the same. Should probably look to do rename also, but leaving until merging after cli extensions PR has gone in